### ISSUE TITLE
Explicitly log the Boavizta API URL used by the scanner

### ISSUE THREAD

**Issue by demeringo** [Boavizta/cloud-scanner#155]
## Problem

When passing a custom URL (`-b` option) to the scanner, there is no easy way to be sure of what is used by the scanner.

## Solution

Log explicitly the URL used when calling the Boavizta API.

**Comment by demeringo** [comment]
We can get the information by using a -vv flag:

e.g.:
```sh
docker run -it -p 8000:8000 -v /etc/ssl/certs/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt -v $HOME/.aws/credentials:/root/.aws/credentials:ro -e ROCKET_ADDRESS=0.0.0.0 -e ROCKET_PORT=8000 -e AWS_PROFILE=$AWS_PROFILE ghcr.io/boavizta/cloud-scanner-cli:latest -vv -b https://dev.api.boavizta.org serve

cloud_scanner_cli: Using default region: eu-west-1
cloud_scanner_cli: Using API at:  https://dev.api.boavizta.org
cloud_scanner_cli: Starting metric server.
rocket::config::config: 🔧 Configured for release.
rocket::config::config: address: 0.0.0.0
rocket::config::config: port: 8000
rocket::config::config: workers: 4
rocket::config::config: ident: Rocket
rocket::config::config: limits: bytes = 8KiB, data-form = 2MiB, file = 1MiB, form = 32KiB, json = 1MiB, msgpack = 1MiB, string = 8KiB
rocket::config::config: temp dir: /tmp
rocket::config::config: http/2: true
rocket::config::config: keep-alive: 5s
rocket::config::config: tls: disabled
rocket::config::config: shutdown: ctrlc = true, force = true, signals = [SIGTERM], grace = 2s, mercy = 3s
rocket::config::config: log level: critical
rocket::config::config: cli colors: true
rocket::rocket: 📬 Routes:
rocket::rocket: (index) GET /
rocket::rocket: (metrics) GET /metrics?<aws_region>
rocket::fairing::fairings: 📡 Fairings:
rocket::fairing::fairings: Shield (liftoff, response, singleton)
rocket::shield::shield: 🛡️ Shield:
rocket::shield::shield: X-Frame-Options: SAMEORIGIN
rocket::shield::shield: X-Content-Type-Options: nosniff
rocket::shield::shield: Permissions-Policy: interest-cohort=()
rocket::rocket: 🚀 Rocket has launched from http://0.0.0.0:8000

```

**Comment by demeringo** [comment]
We should also add a section about the -vv flag in the doc, it is very helpful for debugging.

**Comment by demeringo** [comment]
Closed by #159
